apps/video/index.html

```diff
@@ -65,59 +65,73 @@ <h1 id="thumbnails-number-selected"></h1>
 
         <!-- This element gets positioned over different view elements -->
         <!-- It gets styled differently depending on what it is over -->
-        <ul id="thumbnails"></ul> <!-- Thumbnails inserted here -->
+        <ul role="presentation" id="thumbnails"></ul> <!-- Thumbnails inserted here -->
```

Comment:
I think it would be helpful to include a comment regarding the "presentation" role and why it's being used here.